### ISSUE TITLE
Elements with insufficient color contrast

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/rtp#34]
- [ ] Follow Us text + icons in global "stay connected" section (expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.25:1. Recommendation: change background to #243036.)
- [ ] Follow Us icons in global footer (expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.66:1. Recommendation: change background to #797675.)
